src/System.Net.Security/tests/FunctionalTests/System.Net.Security.Tests.csproj

```diff
@@ -125,23 +126,20 @@
       <Link>Common\Interop\Unix\System.Net.Security.Native\Interop.GssBuffer.cs</Link>
     </Compile>
     <Compile Include="$(CommonPath)\Microsoft\Win32\SafeHandles\GssSafeHandles.cs">
-       <Link>Common\Microsoft\Win32\SafeHandles\GssSafeHandles.cs</Link>
+      <Link>Common\Microsoft\Win32\SafeHandles\GssSafeHandles.cs</Link>
     </Compile>
     <Compile Include="$(CommonPath)\Interop\Unix\System.Net.Security.Native\Interop.NetSecurityNative.cs">
       <Link>Common\Interop\Unix\System.Net.Security.Native\Interop.NetSecurityNative.cs</Link>
     </Compile>
   </ItemGroup>
-  
+  <ItemGroup>
+    <SupplementalTestData Include="$(PackagesDir)System.Net.TestData\1.0.0-prerelease\content\**\*.*" />
+  </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\src\System.Net.Security.csproj">
-      <Project>{89F37791-6254-4D60-AB96-ACD3CCA0E771}</Project>
+      <Project>{89f37791-6254-4d60-ab96-acd3cca0e771}</Project>
       <Name>System.Net.Security</Name>
-      <!-- tests build as linux/osx but product binary is unix -->
-      <OSGroup Condition="'$(TargetsUnix)' == 'true'">Unix</OSGroup>
```

Comment:
Without re-adding the reference from VS I wasn't able to debug product code. (Doing that from UI caused the rest of formatting changes.)